packages/common-types/src/accountLabels.ts

```diff
@@ -1,4 +1,4 @@
-export type AccountLabels = Record<string, string>;
+export type AccountLabels = Record<string, { label: string; lastModified?: number }>;
```

Comment:
Why is timestamp optional?

Reply:
A lot of the functions used AccountLabels for posting a label, and only used the `label` part, lastModified was not required for those

Reply:
I personally find it confusing to make it optional so that you can use it for both reading and writing. I would personally use two different types in that case, but I'll leave it to you 😄